src/components/libraryHeader1.tsx

```diff
@@ -0,0 +1,118 @@
+import styled from '@emotion/styled';
+import { Link } from 'react-router-dom';
+
+export default function LibraryHeader1() {
+  return (
+    <Header>
+      <LeftContainer>
+        <Link to={'/'} style={{ textDecoration: 'none' }}>
+          <LogoContainer>
+            <LogobarIcon src="long Black bar.svg" alt="logo_blackbar"></LogobarIcon>
+            <LogoTitle>brunch story</LogoTitle>
+          </LogoContainer>
+        </Link>
+
+        <PageLinker>최근 업로드 글</PageLinker>
+        <ClickedPageLinker>최근 읽었던 글</ClickedPageLinker>
+        <PageLinker>내가 구독한 글</PageLinker>
+      </LeftContainer>
+
+      <RightContainer>
+        <WriteButton>
+          <WriteContent>글쓰기</WriteContent>
+        </WriteButton>
+        <ProfileImg />
+        <SearchIcon src="icn_search.png" alt="searchIcon"></SearchIcon>
```

Comment:
아이콘을 추가하는건 내가 올려놓은 PR 확인하면 좋을 것 같아! 어제 새벽에 급하게 올린거라 확인 못했을 수도 있어서 천천히 확인해보고 수정해줘! 수정 방법은

```javascript
import { Icon } from '@components/index';
import { icons } from '@styles/icons';
```
위와 같이 import 해주고,
```suggestion
        <Icon icon={icons.search} />
```
이렇게 사용하면 돼!

지금은 search 아이콘이 추가돼있어서 바로 사용할 수 있는데, 또 새로운 아이콘을 추가하고싶다면 styles/icons.ts 파일에 추가하고 쓰면 돼!

Reply:
넵 수정했습니다~ 😄